README.md

```diff
@@ -13,9 +13,9 @@ applications' configurations, leveraging easy application integration.
 
 ## Often used links ##
 
-- [build server](http://build.libelektra.org:8080/)
-- [tutorials](http://git.libelektra.org/blob/master/doc/tutorials/)
-- [API docu](http://doc.libelektra.org/api/latest/html/)
+- [build server](https://build.libelektra.org/)
```

Comment:
i slowly suspect that the github desktop app did all kinds of messy things. I've switched to sourcetree now and it think its much better, gives you a much better indication of what you are about to do and its very easy to rebase or work with branches. I just created a new branch based on the last good commit of super-ls, so no cherry picking needed. Will open a new pull request after dealing with the last few things here.